src/main/resources/Transportation_Network_Providers_reduced_records.csv

```diff
@@ -1,4 +1,4 @@
-﻿Trip ID,TripStartTimestamp,TripEndTimestamp,TripSeconds,TripMiles,PickupCensusTract,DropoffCensusTract,PickupCommunityArea,DropoffCommunityArea,Fare,Tip,AdditionalCharges,TripTotal,SharedTripAuthorized,TripsPooled,PickupCentroidLatitude,PickupCentroidLongitude,PickupCentroidLocation,DropoffCentroidLatitude,DropoffCentroidLongitude,DropoffCentroidLocation
+﻿TripID,TripStartTimestamp,TripEndTimestamp,TripSeconds,TripMiles,PickupCensusTract,DropoffCensusTract,PickupCommunityArea,DropoffCommunityArea,Fare,Tip,AdditionalCharges,TripTotal,SharedTripAuthorized,TripsPooled,PickupCentroidLatitude,PickupCentroidLongitude,PickupCentroidLocation,DropoffCentroidLatitude,DropoffCentroidLongitude,DropoffCentroidLocation
```

Comment:
Current records in main/resources should be removed - we don't want to bundle them in the JAR.